GAE/src/com/gallatinsystems/survey/dao/QuestionDao.java

```diff
@@ -124,7 +124,7 @@ public void delete(List<Question> qList) {
      * @param question
      */
     public void delete(Question question) throws IllegalDeletionException {
-        delete(question, Boolean.TRUE);
+        delete(question, Boolean.FALSE);
```

Comment:
Both occasions that this `delete(question, boolean)` is called, we set the boolean to `false`. Maybe we just remove the part of the method that readjusts the order, along with the parameter?

Reply:
Just though it was shame do remove a capability, in case we ever do non-interactive question removal.

Reply:
I thought we should always keep it interactive to avoid surprises to the user.  Thats why if possible we should move much of the logic that affects how the user sees things to the frontend.